### DISCUSSION TITLE
Is the desktop client fully open source?

### DISCUSSION THREAD

**Discussion by trymeouteh** [hoppscotch/hoppscotch#4168] [Q&A] [answered]
I realized the builds of the desktop app are in this repo with no source code available.

https://github.com/hoppscotch/releases

When the desktop client is officially released, will it be fully open source?

**Comment by ruizlenato** [comment]
up

**Reply by trymeouteh** [reply]
Do you mean "yup"?

**Comment by liyasthomas** [comment, marked as answer]
Here's the source code: https://github.com/hoppscotch/hoppscotch/tree/main/packages/hoppscotch-selfhost-desktop

**Reply by trymeouteh** [reply]
Thank you!

I would also advise updating the app on AlternativeTo to include Windows, Mac and Linux as a platform

https://alternativeto.net/software/postwoman/about/
